### PR TITLE
Slider control application examples update

### PR DESCRIPTION
**Pull request by cre8ivepark**
Overview
---
Added examples of using Slider control's value.  

### Change color
MRTK_Slider_ColorsExample

Changes
---
- Updated SliderExample scene
- Added example scripts that use Slider control's value (SliderRotateObject.cs, SliderChangeColor.cs, SliderScaleObject.cs)
- Fix bug where slider endpoints would update as mouse moves.
- Expose .Pointer field in SliderEventData so people can know which pointer is manipulating the slider.

Requires
---
This PR requires @julenka 's PR #4022. It should be merged first. Separated PR for easier review.

### CODE REVIEW

**Comment by julenka** [comment]
Hi Yoon, the sliders PR #4022 is in! I changed the public fields of the start and end points of the sliders, as well as updated some of the Animation state machines, so you will probably need to re-import the prefabs. Please see the pinchslider example scene to see examples of the new sliders.

Thank you again for making this great example!

**Comment by cre8ivepark** [comment]
Resolved scene merge conflict. Thanks!

**Comment by Railboy** [comment]
This looks fantastic.

It might be worth changing the thumb sliders on one example to show how easy it is. I swapped in platonic meshes (which are much larger than the slider thumb mesh) and the scale / color animations still worked perfectly after I scaled the thumb transform.

Capture

Nitpick - perhaps we should disable the hand mesh / hand joint buttons on platforms where these have no effect?

**Comment by cre8ivepark** [comment]
Polished the value label typography.
Removed Rotate/Scale example per Nick's feedback - we recommend direct manipulation for the rotate/scale, based on our learnings from previous projects.
MRTK_Slider_ColorRev

MRTK_Slider_LunarModuleLanding

File: Assets/MixedRealityToolkit.Examples/Demos/UX/Slider/Scripts/SliderRotateObject.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    public class SliderRotateObject : MonoBehaviour
```
Comment (julenka) [inline Assets/MixedRealityToolkit.Examples/Demos/UX/Slider/Scripts/SliderRotateObject.cs]: I think we no longer need this.

File: Assets/MixedRealityToolkit.Examples/Demos/UX/Slider/Scripts/SliderScaleObject.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    public class SliderScaleObject : MonoBehaviour
```
Comment (julenka) [inline Assets/MixedRealityToolkit.Examples/Demos/UX/Slider/Scripts/SliderScaleObject.cs]: I think we no longer need this.

**Comment by julenka** [comment]
Yoon, do you still have the rotate and scale examples in the scenes? I thought we wanted those removed.

**Pull request merged**